sdclientapi/__init__.py

```diff
@@ -196,7 +202,11 @@ def authenticate(self) -> bool:
             "one_time_code": self.totp,
         }
 
-        token = requests.post(self.server + "api/v1/token", data=json.dumps(user_data))
+        token = requests.post(
+            self.server + "api/v1/token",
```

Comment:
Should use `urljoin`

Reply:
This is actually the old code, I will have to rebase the PR.